satpy/readers/ghrsst_l2.py

```diff
@@ -61,6 +59,13 @@ def __init__(self, filename, filename_info, filetype_info, engine=None):
         self.filename_info['end_time'] = datetime.strptime(
             self.nc.stop_time, '%Y%m%dT%H%M%SZ')
 
+    @staticmethod
+    def _is_sst_file(name):
+        """Check if file in the tar archive is a valid SST file."""
+        if name.endswith('nc') and 'GHRSST-SSTskin' in name:
+            return True
+        return False
```

Comment:
You can get rid of the coverage miss by returning `name.endswith('nc') and 'GHRSST-SSTskin' in name` :slightly_smiling_face: